start.py

```diff
@@ -0,0 +1,36 @@
+import sys
+import os
+
+HELP_STR = """Syntax:
+    python ./start.py action mcVersion metaUrl
+Example:
+    python ./start.py yarn 1.8.9
+    python ./start.py yarn 1.7.10 https://meta.legacyfabric.net
+    python ./start.py publishlocal 1.8.9"""
+
+def main():
+    args = sys.argv
+    if len(set(["--help", "-h"]) & set(args))>0:
+        print(HELP_STR)
+        return
+    if len(args) < 2:
+        print("Action wasn't specified, input should look like")
+        print(HELP_STR)
+        return
+    if len(args) < 3:
+        print("Version wasn't specified, input should look like:")
+        print(HELP_STR)
+        return
+    action = args[1]
+    version = args[2]
+    if len(args) == 4:
+        os.environ['LEGACY_FABRIC_META_URL'] = args[3]
+    if action == 'yarn':
+        os.environ['MC_VERSION'] = version
+        os.system('gradlew.bat yarn')
```

Comment:
Inconsistency, why does this use the `.bat` version while the other doesn't?
Both should be the same, and based off the user's running OS as Windows requires the `.bat` and unix needs the one without

Reply:
fixed in ead1491